### ISSUE TITLE
ciw.set_seed

### ISSUE THREAD

**Issue by geraintpalmer** [geraintpalmer/Ciw#70]
Ciw uses two random number streams, one from the random module and one from numpy.random module. Therefore in order to replicate results users need to:

    from random import seed
    from numpy import random
    seed(5)
    random.seed(5)

I would rather this be wrapped up in ciw, so user's simply need to:

    ciw.set_seed(5)

I already have this defined and using it in the test suite only. @drvinceknight you mentioned you didn't like this idea, but can't remember why? Any thoughts?

**Comment by drvinceknight** [comment]
> @drvinceknight you mentioned you didn't like this idea

Apologies if I did: I wonder why I thought that. I think it's a good idea. Actually just got a PR opened for Axelrod to do the same for the same reason. Take a look to see the various things I implemented there https://github.com/Axelrod-Python/Axelrod/pull/653:

1. Tests to show that if you do the same thing twice with the new seed function it gives the same result.
2. I called it `axelrod.seed` not `axelrod.set_seed` (which is what I called it at first so when we spoke about this something must have stuck in my head). Rational being that `random.seed` and `numpy.random.seed` imply that it should be `axelrod.seed`, so I think here it should be `ciw.seed`.
3. Documentation about setting the seed.
